### DISCUSSION TITLE
Code splitting

### DISCUSSION THREAD

**Discussion by DiachenkoIgor** [parcel-bundler/parcel#6435] [Q&A]
I want to use lazy load (code splitting) with dynamic import in parcel, however I receive error if file doesn't exist in directory:
`Cannot resolve dependency '/test.js'`

Code for invoking look like:
`ready.on("ready", onReady => {import("/test.js")});`

I build with command:
`parcel build --target browser main.ts --bundle-node-modules`

**Comment by mischnic** [comment]
Are you sure you didn't want to write `import("./test.js")`?

**Reply by DiachenkoIgor** [reply]
I have tried both variants.

**Reply by DeMoorJasper** [reply]
Can you create a minimal reproduction for this?

**Reply by DiachenkoIgor** [reply]
I have 2 files inside one directory:
- main.ts (This file have a lot of imports of libraries)
- test.js

There is line which imports 'test.js' inside 'main.ts'.
